src/redux/bookCategories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const bookCata = 'Categories';
+const initialState = {
+  bookName: bookCata,
```

Comment:
- [ ] According to project requirements, you should include an array of categories (initial state: empty).